ethereum/test/unit_tests/executor_proof.spec.ts

```diff
@@ -0,0 +1,58 @@
+import * as hardhat from 'hardhat';
+import { expect } from 'chai';
+import { ExecutorProvingTest, ExecutorProvingTestFactory } from '../../typechain';
+
+
+describe('Executor test', function () {
```

Comment:
Not for this PR, but in the future please write tests in Foundry

Reply:
ack